src/actions/filter.jsx

```diff
@@ -93,6 +94,22 @@ export function toggleFlagFilter( filterName ) {
   }
 }
 
+/**
+* Notifies the application that a filter was added
+*
+* @param {string} filterName which filter was clicked
+* @param {string} filterValue the value of the filter that was clicked
+* @returns {string} a packaged payload to be used by Redux reducers
+*/
+export function addFilter( filterName, filterValue ) {
```

Comment:
A little nitpicky, but we have unit tests for these action creators too.   It's a pretty pointless test since it's just testing redux, and these tests would go away if we migrated to redux-toolkit and used slices which combines the action creators and reducers into a single file.

Reply:
I wasn't totally certain what action you were saying to do here (leave as-is, add addFilter test, or remove removeFilter test), but I've added the test in https://github.com/cfpb/ccdb5-ui/pull/390/commits/2553103062337dfb72088c222aedd67cfbf38753 and we can trim the superfluous ones altogether when you're back.